### PR TITLE
Update getting started docs

### PR DESCRIPTION
**Pull request by samanehsan**
Include a section about creating Terra accounts and getting added to the admin user group.

### CODE REVIEW

File: docs/jade-getting-started.md [hunk 1]
```diff
@@ -52,6 +52,18 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
+### 3. Create Terra Accounts
+The Data Repo uses [Sam](https://github.com/broadinstitute/sam) for identity and access management. To register
+as a new user, create an account through Terra. Use a non-Broad email for the non-prod instances:
```
Comment (fboulnois) [inline docs/jade-getting-started.md]: > Use a non-Broad email for the non-prod instances

I think we should stress that this is an email account that is designed specifically for development purposes as opposed to a personal email account.

File: docs/jade-getting-started.md [hunk 2]
```diff
@@ -52,6 +52,18 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
+### 3. Create Terra Accounts
```
Comment (fboulnois) [inline docs/jade-getting-started.md]: Should this be a subheading of section 2 or a separate section?

**Review by fboulnois — CHANGES_REQUESTED** [review]
A few comments:

File: docs/jade-getting-started.md (hunk 2 above)
Comment (samanehsan) [reply docs/jade-getting-started.md]: Good catch! I meant to make that a separate section.

File: docs/jade-getting-started.md (hunk 1 above)
Comment (snf2ye) [inline docs/jade-getting-started.md]: At least when we were working w/ just the stewards group, we always added everyone on the team to the stewards group. I'm actually not sure if we would want to do that with the new admins group. But, maybe a helpful instruction here would be to "Request for teammate to add your non-broad email to the relevant groups in SAM for each environment".

Comment (snf2ye) [reply docs/jade-getting-started.md]: WHOOPS - didn't see this 5 lines down! Sorry!

**Review by snf2ye — APPROVED** [review]
LGTM - thanks for adding this!

File: docs/jade-getting-started.md [hunk 3]
```diff
@@ -52,7 +52,20 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
-## 3. Install Homebrew
+## 3. Create Terra Accounts
+The Data Repo uses [Sam](https://github.com/broadinstitute/sam) for identity and access management. To register
```
Comment (fboulnois) [inline docs/jade-getting-started.md]:
```suggestion
## 3. Create Terra Accounts

The Data Repo uses [Sam](https://github.com/broadinstitute/sam) for identity and access management. To register
```

**Review by fboulnois — COMMENTED** [review]
Small recommendation:

**Review by fboulnois — APPROVED** [review]
lgtm

**Pull request merged**